### PR TITLE
Cmi/remember/2

### PR DESCRIPTION
**Pull request by AlephNotation**
Adding the ability to recall relevant past action sequences

### CODE REVIEW

File: src/collectiveMemory/remember.ts [hunk 1]
```diff
@@ -9,13 +10,55 @@ export const HDRConfig = z.object({
...
+
+  return data.map((m: any) =>
+    Memory.parse({
+      actionStep: m.action,
+      objectiveState: m.state,
+    })
+  ) as Memory[];
+}
+
 // TODO: fill out
```
Comment (matildepark) [inline src/collectiveMemory/remember.ts:47]: ?

**Review by matildepark — APPROVED** [review]
reads ok, i need to add hdr key to `bin` now

File: src/collectiveMemory/remember.ts (hunk 1 above)
Comment (AlephNotation) [reply src/collectiveMemory/remember.ts:47]: The TS compiler doesn't register this unless I explicitly set the type. I can fix in the morning tho

Comment (matildepark) [reply src/collectiveMemory/remember.ts:47]: the `//TODO: fill out` because you did fill it out

**Pull request merged**